Resources/Views/Authentication/presentation_form.leaf

```diff
@@ -3,29 +3,43 @@
     #export("content"):
         #extend("Shared/_header")
         <div class="row">
-            <form method="POST" action="/api/v1/presentation/create" enctype="multipart/form-data">
-                <h1>Create presentation</h1>
+            #if(presentation):
+              <form method="POST" action="/api/v1/presentations/#(presentation.id)" enctype="multipart/form-data">
```

Comment:
As most of the line is the same, an inline if adding the presentation.id may be better here